app/widget/src/main/java/tt/co/jesses/moonlight/widget/MoonlightWidget.kt

```diff
@@ -0,0 +1,30 @@
+package tt.co.jesses.moonlight.widget
+
+import android.content.Context
+import androidx.compose.runtime.Composable
+import androidx.glance.GlanceId
+import androidx.glance.appwidget.GlanceAppWidget
+import androidx.glance.appwidget.provideContent
+import androidx.glance.layout.Alignment
+import androidx.glance.layout.Column
+import androidx.glance.layout.fillMaxSize
+import androidx.glance.text.Text
+
+class MoonlightWidget : GlanceAppWidget() {
+    override suspend fun provideGlance(context: Context, id: GlanceId) {
+        provideContent {
+            MoonlightWidgetContent()
+        }
+    }
+
+    @Composable
+    fun MoonlightWidgetContent() {
+        Column(
+            modifier = androidx.glance.GlanceModifier.fillMaxSize(),
+            verticalAlignment = Alignment.Vertical.CenterVertically,
+            horizontalAlignment = Alignment.Horizontal.CenterHorizontally
+        ) {
+            Text("Hello World")
```

Comment:
I was hoping to use the same Moonlight screen gradient that is used in the Android and Wear modules